example/tsconfig.base.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
```

Comment:
can a bunch of this stuff be inherited from the base tsconfig in the root directory?